packages/cfd/src/Containers/mt5-compare-table-content.tsx

```diff
@@ -311,7 +311,16 @@ const DMT5CompareModalContent = ({
     }, []);
 
     const getAvailableAccountsContent = (_content: TModalContentProps[]) => {
```

Comment:
Nit: It is not a standard naming convention for an argument.
```suggestion
    const getAvailableAccountsContent = (content: TModalContentProps[]) => {
```